packages/block-editor/src/hooks/align.js

```diff
@@ -207,6 +212,130 @@ export const withDataAlign = createHigherOrderComponent(
 		return <BlockListBlock { ...props } wrapperProps={ wrapperProps } />;
 	}
 );
+export function AlignmentVisualizer( {
+	value = 'none',
+	allowedAlignments,
+	clientId,
+} ) {
+	const layout = useLayout();
+	const blockName = useSelect(
+		( select ) => select( blockEditorStore ).getBlockName( clientId ),
+		[ clientId ]
+	);
+
+	const [ editorWidth, setEditorWidth ] = useState( 0 );
+	const ref = useRefEffect( ( node ) => {
+		const { ownerDocument } = node;
+		const { defaultView } = ownerDocument;
+		const editorWrapperElement = ownerDocument.querySelector(
+			'.editor-styles-wrapper'
```

Comment:
Using a classname is also brittle. Would be good to consider alternatives if there are any.